infrastructure/template.yaml

```diff
@@ -440,6 +440,63 @@ Resources:
       FilterPattern: ""
       LogGroupName: !Ref HelloWorldFunctionLogGroup
 
+ ####################################################################
+#                                                                  #
+# Claimed Identity Function                                        #
+#                                                                  #
+####################################################################
+ 
+ ClaimedIdentityFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: uk.gov.di.ipv.cri.claimedidentity.api.handler.ClaimedIdentityHandler::handleRequest
+      CodeUri: ../src/claimed-identity/
+      Runtime: nodejs16.x
+      Environment:
+        Variables:
+          SESSION_TABLE: 'session-common-cri-api'
+          POWERTOOLS_SERVICE_NAME: !Sub "${CriIdentifier}-claimed-identity"
+      Policies:
+        - AWSLambdaBasicExecutionRole
+        - AWSXrayWriteOnlyAccess
+        - DynamoDBWritePolicy:
+            TableName: !Sub "{{resolve:ssm:/${CommonStackName}/SessionTableName}}"
+        - DynamoDBReadPolicy:
+            TableName: !Sub "{{resolve:ssm:/${CommonStackName}/SessionTableName}}"
+        - DynamoDBReadPolicy:
+            TableName: !Sub "{{resolve:ssm:/${CommonStackName}/PersonIdentityTableName}}"
+        - DynamoDBWritePolicy:
+            TableName: !Sub "{{resolve:ssm:/${CommonStackName}/PersonIdentityTableName}}"
+          - Sid: ReadParametersPolicy
+            Effect: Allow
+            Action:
+              - ssm:GetParameter
+            Resource:
+              - !Sub "arn:aws:ssm:${AWS::Region}:${AWS::AccountId}:parameter/${AWS::StackName}/LookupTableName"
+    Metadata: # Manage esbuild properties
+      BuildMethod: esbuild
+      BuildProperties:
+        Minify: true
+        Target: "es2020"
+        Sourcemap: true # Enabling source maps will create the required NODE_OPTIONS environment variables on your lambda function during sam build
+        EntryPoints:
+        - app.ts
+
+  ClaimedIdentityFunctionLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub "/aws/lambda/${ClaimedIdentityFunction}"
+      RetentionInDays: 30
+
+  ClaimedIdentityFunctionLogsSubscriptionFilterCSLS:
+    Type: AWS::Logs::SubscriptionFilter
+    Condition: IsNotDevEnvironment
+    Properties:
+      DestinationArn: "arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython"
```

Comment:
This will cause issue in AWS build account as we havnt done the integration with splunk